Src/WitsmlExplorer.Frontend/components/ContextMenus/RiskContextMenu.tsx

```diff
@@ -37,6 +38,12 @@ const RiskObjectContextMenu = (props: RiskObjectContextMenuProps): React.ReactEl
         <MenuItem key={"properties"} onClick={onClickModify} disabled={checkedRiskObjectRows.length !== 1}>
           <StyledIcon name="settings" color={colors.interactive.primaryResting} />
           <Typography color={"primary"}>Properties</Typography>
+        </MenuItem>,
+        <MenuItem key={"delete"} onClick={() => onClickDelete(checkedRiskObjectRows, dispatchOperation, dispatchNavigation)} disabled={checkedRiskObjectRows.length === 0}>
```

Comment:
Follow the typical order of having "delete" above "properties", and a divider above "properties"